qlib/kernel/SignalDef.rs

```diff
@@ -112,13 +112,50 @@ pub struct PtRegs {
 #[cfg(target_arch = "aarch64")]
 impl fmt::LowerHex for PtRegs {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "SP:[{:#x}]
+        write!(f, "\nSP:[{:#x}]
                    PC:[{:#x}]
                    State:[{:#x}]
                    X0:[{:#x}]
-                   GenRegs:[{:#?}]",
-                   self.sp, self.pc, self.pstate,
-                   self.orig_x0, self.regs)
+                   GenRegs:[
```

Comment:
just out of curiosity,  can we apply something like `derive(debug)` to shorten this?

Reply:
I don't know. I have it like this because I don't want to count by myself.

Reply:
doesn't "\n" work? why all the newlines?

Reply:
Adjusted.